### DISCUSSION TITLE
Methodology section: poor phrasing in statement about carbon intensity

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#183] [SCI feedback]
"Electricity has a carbon intensity depending on where and when it is consumed" - 
This is wrong: carbon intensity is related to the generation of electricity.

**Comment by atg-abhishek** [comment]
Agreed @dschien - once the PRs today are merged, I believe that this statement won't be there in the SCI but if it is, please feel free to create a new PR to address this. Thanks!
